src/scan/postgres_scan.cpp

```diff
@@ -44,11 +44,11 @@ PostgresScanGlobalState::InitGlobalState(duckdb::TableFunctionInitInput &input)
 
 	if (input.CanRemoveFilterColumns()) {
 		for (duckdb::idx_t i = 0; i < input.projection_ids.size(); i++) {
```

Comment:
Should this be changed to column_ids too?
```suggestion
		for (duckdb::idx_t i = 0; i < input.column_ids.size(); i++) {
```

Reply:
Turns out the answer is no

Reply:
In this branch, we need to use projection_ids list.